modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Options/Options.es.js

```diff
@@ -316,21 +313,15 @@ const Main = ({
 				onChange={(value) => onChange({}, value)}
 				value={value}
 			>
-				{({
-					defaultOptionRef,
-					handleBlur,
-					handleField,
-					index,
-					option,
-				}) => (
+				{({defaultOptionRef, handleField, index, option}) => (
 					<KeyValue
 						generateKeyword={option.generateKeyword}
 						keyword={option.value}
 						keywordReadOnly={
 							defaultLanguageId !== editingLanguageId
 						}
 						name={`option${index}`}
-						onBlur={handleBlur}
+						onBlur={() => {}}
```

Comment:
@carolmariaabb, if there are no more handlers, what it's the purpose of keeping the prop `onBlur`?